fedimint-load-test-tool/src/common.rs

```diff
@@ -0,0 +1,393 @@
+    let operation_id = OperationId::new_random();
+    for _ in 0..quantity {
+        let output = mint_client
+            .create_output(&mut module_transaction, operation_id, 1, denomination)
+            .await;
+        tx = tx.with_output(output.into_dyn(client_module_instance.id));
+    }
+    drop(module_transaction);
+    let operation_meta_gen = |_txid, _outpoint| ();
+    let txid = client
+        .finalize_and_submit_transaction(
+            operation_id,
+            MintCommonGen::KIND.as_str(),
+            operation_meta_gen,
+            tx,
+        )
+        .await?;
+    let tx_subscription = client.transaction_updates(operation_id).await;
+    tx_subscription.await_tx_accepted(txid).await?;
+    dbtx.commit_tx().await;
+    for i in 0..quantity {
+        let out_point = OutPoint {
+            txid,
+            out_idx: i as u64,
+        };
+        mint_client
+            .await_output_finalized(operation_id, out_point)
+            .await?;
+    }
+    Ok(())
+}
+
+pub struct FedimintCli;
+impl ToCmdExt for FedimintCli {
+    type Fut = std::future::Ready<devimint::util::Command>;
+
+    fn cmd(self) -> Self::Fut {
+        // try to use alias if set
+        let fedimint_cli = std::env::var("FM_MINT_CLIENT")
+            .map(|s| s.split_whitespace().map(ToOwned::to_owned).collect())
+            .unwrap_or_else(|_| vec!["fedimint-cli".into()]);
+        let mut cmd = tokio::process::Command::new(&fedimint_cli[0]);
+        cmd.args(&fedimint_cli[1..]);
+        std::future::ready(devimint::util::Command {
+            cmd,
+            args_debug: fedimint_cli,
+        })
+    }
+}
+
+pub struct LnCli;
+impl ToCmdExt for LnCli {
+    type Fut = std::future::Ready<devimint::util::Command>;
+
+    fn cmd(self) -> Self::Fut {
+        // try to use alias if set
+        let lncli = std::env::var("FM_LNCLI")
+            .map(|s| s.split_whitespace().map(ToOwned::to_owned).collect())
+            .unwrap_or_else(|_| vec!["lncli".into()]);
+        let mut cmd = tokio::process::Command::new(&lncli[0]);
+        cmd.args(&lncli[1..]);
+        std::future::ready(devimint::util::Command {
+            cmd,
+            args_debug: lncli,
+        })
+    }
+}
+
+pub struct ClnLightningCli;
+impl ToCmdExt for ClnLightningCli {
+    type Fut = std::future::Ready<devimint::util::Command>;
+
+    fn cmd(self) -> Self::Fut {
+        // try to use alias if set
+        let lightning_cli = std::env::var("FM_LIGHTNING_CLI")
+            .map(|s| s.split_whitespace().map(ToOwned::to_owned).collect())
+            .unwrap_or_else(|_| vec!["lightning-cli".into()]);
+        let mut cmd = tokio::process::Command::new(&lightning_cli[0]);
+        cmd.args(&lightning_cli[1..]);
+        std::future::ready(devimint::util::Command {
+            cmd,
+            args_debug: lightning_cli,
+        })
+    }
+}
+
+pub struct GatewayClnCli;
+impl ToCmdExt for GatewayClnCli {
+    type Fut = std::future::Ready<devimint::util::Command>;
```

Comment:
BTW, you don't need to impl this trait
```suggestion
impl GatewayClnCli {
    pub async fn cmd(&self) -> devimint::util::Command;
```